test/indentation_test.js

```diff
@@ -0,0 +1,32 @@
+/*jshint node: true*/
+
+var fs = require("fs");
+var assert = require("assert");
+
+var hbsfy = require("hbsfy");
+var Handlebars = require("hbsfy/runtime");
+
+
+var templatePath = __dirname + "/indentation.hbs";
+var exported = __dirname + "/compiled.js";
+
+try {
+  fs.unlinkSync(exported);
+} catch (err) { }
+
+fs.createReadStream(templatePath)
+.pipe(hbsfy(templatePath, {
+  processContent: function(content) {
+    return content
+      .replace(/^[\x20\t]+/mg, '')
+      .replace(/[\x20\t]+$/mg, '')
+      .replace(/[\r\n]/g, '');
```

Comment:
I know this is just the test, but isn't this kind of transformation dangerous? What about `<pre>` tags?